### PR TITLE
Remove from/import and update to PHP5.4

### PR DESCRIPTION
**Pull request by Hywan**

### CODE REVIEW

File: Transport/Smtp.php [hunk 1]
```diff
@@ -335,6 +317,11 @@ public function send ( \Hoa\Mail\Message $message ) {
     /**
      * H-MAC.
```
Comment (vonglasow) [inline Transport/Smtp.php]: Ok to read the RFC but a little description about what is H-MAC could be useful

**Comment by vonglasow** [comment]
:+1: Just make a code review, not tested

File: Transport/Smtp.php (hunk 1 above)
Comment (Hywan) [inline Transport/Smtp.php]: This is a well-known cryptographic function, check http://php.net/hash_hmac :-). What would you like to precise?

Comment (vonglasow) [inline Transport/Smtp.php]: oki I didn't remember this function so just what you just said ;) "a well-known cryptographic function" ;)

**Pull request merged**